### PR TITLE
delete lint from gitlab build

### PR DESCRIPTION
**Pull request by ilaner**
Delete lint from gitlab build. Keep pre-commit only

### CODE REVIEW

**Comment by dorschw** [comment]
[nightly](https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/pipelines/946759)

File: .gitlab/ci/.gitlab-ci.global.yml [hunk 1]
```diff
@@ -438,69 +440,7 @@
         ./Tests/scripts/sdk_pylint_check.sh
       fi
     - section_end "Test Infrastructure"
-    - rm -rf Packs/MyPack # remove MyPack from the Packs folder, workaround until demisto-sdk 1.27.5 will be released
```
Comment (kobymeir) [inline .gitlab/ci/.gitlab-ci.global.yml:441]: at line 400 rename `.unittests-and-lint-settings` to `.pre-commit-settings`

**Review by kobymeir — CHANGES_REQUESTED** [review]
please consult with native image owner on the `test-native-candidate:run-lint-with-native-candidate` build and also changing it to pre-commit.

**Review by dorschw — COMMENTED** [review]
Let's remove it from the CircleCI step too, it's running in GitHub anyway.

File: .gitlab/ci/.gitlab-ci.global.yml (hunk 1 above)
Comment (ilaner) [reply .gitlab/ci/.gitlab-ci.global.yml:441]: renamed

**Review by kobymeir — APPROVED** [review]

**Comment by ilaner** [comment]
> Let's remove it from the CircleCI step too, it's running in GitHub anyway.

We will remove it in another PR.
It will require force merge and deleting the circle ci from github settings

**Pull request merged**